Makefile

```diff
@@ -3,59 +3,63 @@ LOCALEPREFIX ?= $(PREFIX)/share/locale
 MANPREFIX    ?= $(PREFIX)/share/man
 PANDOC       ?= $(shell which pandoc)
 XDGPREFIX    ?= /etc/xdg
+SCRIPTS      ?= $(shell find ./bin -type f -executable -printf "%f\n")
 
 .PHONY: setup
 setup:
 	command -v cram   || aurget cram
 	command -v vbump  || aurget vbump-git
 	command -v pandoc || stack install pandoc
 
-locale/downgrade.pot: bin/downgrade
+locale/%.pot: bin/%
 	xgettext \
 		--from-code=utf-8 -L shell \
-		--package-name=downgrade \
+		--package-name=$* \
 		--copyright-holder=$(AUTHOR) \
 		-o $@ $<
-	find ./locale/*po -exec msgmerge --update {} $@ \;
+	find ./locale/$*/*.po -exec msgmerge --update {} $@ \;
```

Comment:
Could use `-name '*.po'`, which is a touch clearer IMO -- but functions the same.

Reply:
True, modified